spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/support/EntityGraphFactory.java

```diff
@@ -31,6 +31,7 @@
  * @author Jens Schauder
  * @author Petr Strnad
  * @author Greg Turnquist
+ * @author Jinho Lee
```

Comment:
Trivial changes and especially minor removals are not attributed. Happy to merge your PR once you remove the added `@author` tags.

Reply:
```@author``` removed. Thank you